### DISCUSSION TITLE
Does anyone encounter the same issues on Isaac Sim when running demos?

### DISCUSSION THREAD

**Discussion by xueminchi** [NVlabs/curobo#89] [Software Q&A]
![Screenshot from 2023-12-20 11-29-57](https://github.com/NVlabs/curobo/assets/39512956/c6e6a91a-ee44-46a8-84cf-5bb79d7e93e7)

Everything works fine, but the robot just cannot move when I try demos in Isaac Sim. The terminal kept showing updated world, but nothing happens in Isaac Sim. 

I'm not sure if this is related to some potential issues I haven't resolved. 
```
nvidia-srl-usd 0.9.0 requires usd-core<24.00,>=21.11, which is not installed.
nvidia-srl-usd-to-urdf 0.3.2 requires usd-core<24.00,>=21.11, which is not installed.
gymnasium 0.28.1 requires cloudpickle>=1.2.0, which is not installed.
gymnasium 0.28.1 requires farama-notifications>=0.0.1, which is not installed.
gymnasium 0.28.1 requires jax-jumpy>=1.0.0, which is not installed.
aiobotocore 1.2.0 requires botocore<1.19.53,>=1.19.52, but you have botocore 1.20.49 which is incompatible.
```
Since these issues don't bother running the demos, I've ignored them for the moment.

**Comment by balakumar-s** [comment]
What is printed when you move the red cube?

**Comment by xiaoQ0077** [comment]
I have the same problem when installing _omniisaacgymenvs_, it shows that there are some modules that are not installed, does this affect the later? How did you solve this problem? Thanks a lot!

**Reply by balakumar-s** [reply]
Can you share a screen shot?

**Reply by xiaoQ0077** [reply]
Hi, balakumar-s. 
I'm having problems installing _omniisaacgymencs_ as shown below, which shows that **nvidia-srl-usd 0.13.0, nvidia-srl-usd-to-urdf 0.5.0, gymnasium 0.28.1** were not installed successfully, and I've seen in other forums that it's possibly a network problem, so I pip it again and the results shown are in the second image, I'm not sure if this solved the problem. I did report an error when I ran the _task=FrankaCabinet_ example and it wouldn't run (figure 3). If you could take a look at this for me, it would be much appreciated!
![image](https://github.com/NVlabs/curobo/assets/131855153/7c93cdd0-f0e0-487e-9ff7-60fb17e30b00)
![image](https://github.com/NVlabs/curobo/assets/131855153/0f091eec-eb0e-4611-b593-f596aec7f177)
![}M@$X)(~{%VM1 I4JX6PZX](https://github.com/NVlabs/curobo/assets/131855153/c0298a34-22de-475c-9b23-00576b5830f3)

**Comment by JenniferBuehler** [comment]
I did install `usd-core` due to the same message, and took a wild shot at the version by using 23.11. I have other issues with the demos though, see my response to #303
